app/src/main/java/com/github/se/studybuddies/MainActivity.kt

```diff
@@ -143,6 +145,16 @@ class MainActivity : ComponentActivity() {
                 Log.d("MyPrint", "Successfully navigated to TimerScreen")
               }
             }
+            composable(
+                route = "${Route.SHAREDTIMER}/{groupUID}",
+                arguments = listOf(navArgument("groupUID") { type = NavType.StringType })) {
+                    backStackEntry ->
+                  val groupUID = backStackEntry.arguments?.getString("groupUID")
+                  if (groupUID != null) {
+                    SharedTimerScreen(navigationActions, SharedTimerViewModel(groupUID))
+                    Log.d("MyPrint", "Successfully navigated to GroupScreen")
```

Comment:
maybe change the Lod.d message to SaredTimerScreen instead of GroupScreen

Reply:
done

Reply:
the button now can subtract